crates/algokit_transact/src/transactions/asset_config.rs

```diff
@@ -164,7 +164,7 @@ pub struct AssetConfigTransactionFields {
     ///
     /// If not set or set to the Zero address the asset becomes permanently immutable.
     #[builder(default)]
-    pub manager: Option<Address>,
+    pub manager: Option<Account>,
```

Comment:
Curious why the type was changed? A manager could be a multisig? Applies to all of these.